src/commands/flow/internal/allFlowsCompleted.ts

```diff
@@ -0,0 +1,34 @@
+export const allFlowsCompleted = (
+  flows: any
+): {
+  completed: boolean
+  completedWithAllSuccess: boolean
+} => {
+  const flowNames = Object.keys(flows)
+
+  let jobsCount = 0
+  let jobsWithSuccessStatus = 0
+  let jobsWithNotSuccessStatus = 0
```

Comment:
please rename variable to `jobsWithFailureStatus`

Reply:
fixed